packages/wallet/backend/src/account/service.ts

```diff
@@ -132,14 +129,18 @@ export class AccountService implements IAccountService {
       user.rapydWalletId
     )
 
-    accounts.forEach((acc) => {
-      acc.balance = transformBalance(
+    for (let i = 0; i < accounts.length; i++) {
+      accounts[i].balance = transformBalance(
         accountsBalance.data?.find(
-          (rapydAccount) => rapydAccount.currency === acc.assetCode
+          (rapydAccount) => rapydAccount.currency === accounts[i].assetCode
         )?.balance ?? 0,
-        acc.assetScale
+        accounts[i].assetScale
       )
-    })
+      accounts[i].paymentPointers = await PaymentPointer.query().where(
```

Comment:
on getting accounts the payment pointers can be received using `withGraphFetched`

also the get accounts should be moved after the check that user exists and has a wallet id (not related to your PR but it will be good to get that change as well)

Reply:
@raducristianpopa is the frontend expecting the list account to return the payment pointers as well?

Reply:
>is the frontend expecting the list account to return the payment pointers as well?

Yes, I was thinking about a separate endpoint but this works as well.

Reply:
@raducristianpopa as on developer keys view there is no need for balance (no need to request rapyd) and on other pages there is not need for all the payment pointers, as they are listed separately for selected account - I think it makes more sense to have a separate endpoint